### PR TITLE
Do not install software-properties-common during bootstrap for Ubuntu Focal

### PR DESCRIPTION
**Pull request by nickygerritsen**
I wonder if we need the package at all? At least it now builds for me on Focal.

### CODE REVIEW

**Comment by mvr320** [comment]
It seems to be a helper for apt (https://askubuntu.com/questions/1000118/what-is-software-properties-common) I think when you're building the chroot you're aware how to do things without the help of apt in worst case.

So I would say this is actually an edge case which should not be needed?

**Comment by nickygerritsen** [comment]
> It seems to be a helper for apt (https://askubuntu.com/questions/1000118/what-is-software-properties-common) I think when you're building the chroot you're aware how to do things without the help of apt in worst case.
> 
> So I would say this is actually an edge case which should not be needed?

Yes that's what I also think. Maybe @eldering or @thijskh have thoughts? If they also agree I'd suggest we remove the whole thing.

**Comment by thijskh** [comment]
Can be safely removed

**Comment by nickygerritsen** [comment]
OK I removed it completely

File: misc-tools/dj_make_chroot.in [hunk 1]
```diff
@@ -159,7 +159,7 @@ fi
 if [ "$DISTRO" = 'Ubuntu' ]; then
 
 # Packages to include during bootstrap process (comma separated):
-INCLUDEDEBS="software-properties-common"
+INCLUDEDEBS=""
 
 # Packages to install after upgrade (space separated):
 INSTALLDEBS="default-jre-headless pypy python3 locales"
```
Comment (mvr320) [inline misc-tools/dj_make_chroot.in:165]: If you're removing, should pypy still be there and should python3->pypy3?

Comment (nickygerritsen) [reply misc-tools/dj_make_chroot.in:165]: Good question. We might want to replace both with pypy3 indeed, but let's leave that for a different PR.

**Review by thijskh — APPROVED** [review]

**Pull request merged**